server.go

```diff
@@ -30,8 +31,12 @@ func getDB() (db *sql.DB) {
 }
 
 func main() {
-	http.HandleFunc("/", handleIndex)
-	http.HandleFunc("/steam/discounts", handleSteamDiscounts)
+    mux := mux.NewRouter()
```

Comment:
btw why we need `mux` from gorilla? personally speaking for our use case we don't need a full web framework?